assets/js/base/components/product-price/index.tsx

```diff
@@ -191,17 +191,17 @@ export interface ProductPriceProps {
 	/**
 	 * Currency configuration object
 	 */
-	currency: Currency | Record< string, never >;
+	currency?: Currency | Record< string, never >;
 	/**
 	 * The string version of the element to use for the price interpolation
 	 *
 	 * **Note:** It should contain `<price/>` (which is also the default value)
 	 */
-	format: string;
+	format?: string;
 	/**
 	 * The current price
 	 */
-	price: number | string;
+	price?: number | string;
```

Comment:
This part had been changed to prevent a TS error in the following sections:
- https://github.com/woocommerce/woocommerce-blocks/blob/8e8b4a003d0d95c98b0cb388d034a0b73709f1e2/assets/js/atomic/blocks/product-elements/price/block.tsx#L46
- https://github.com/woocommerce/woocommerce-blocks/blob/8e8b4a003d0d95c98b0cb388d034a0b73709f1e2/assets/js/atomic/blocks/product-elements/price/block.tsx#L59-L76

Reply:
A similar issue I mentioned in [this comment](https://github.com/woocommerce/woocommerce-blocks/pull/7683#discussion_r1024804540) can be seen on this line:

https://github.com/woocommerce/woocommerce-blocks/blob/ae10c57e525e1d14b3f83689789765a298e66793/assets/js/base/components/product-price/index.tsx#L287

Reply:
Looks good to me 🙌🏻